client/repo.c

```diff
@@ -340,21 +340,25 @@ TDNFGetGPGSignatureCheck(
             nGPGSigCheck = pRepo->nGPGCheck;
             if(nGPGSigCheck)
             {
-                if (IsNullOrEmptyString(pRepo->pszUrlGPGKey))
-                {
-                    dwError = ERROR_TDNF_NO_GPGKEY_CONF_ENTRY;
+                if (ppszUrlGPGKey != NULL) {
```

Comment:
Use
```
if ()
{
   ...
}
```

Reply:
request here, this function has like 5 indentation levels, can you please rewrite this one and not exceed 3 levels?
I kinda agree with kernel coding guide line:
```
if you need more than 3 levels of indentation, you’re screwed anyway, and should fix your program.
```

Reply:
I reduced it by one level. I don't think trying more would improve readability.